go.mod

```diff
@@ -39,6 +39,7 @@ require (
 )
 
 require (
+	github.com/algorand/sortition v0.0.0-20221122025755-543ffac1d177 // indirect
```

Comment:
This should probably be a direct dependency, I'm not sure why it always does this.